tests/conftest.py

```diff
@@ -170,8 +170,7 @@ def full_record(users):
             "version": "v1.0",
             "rights": [{
                 "title": "Creative Commons Attribution 4.0 International",
-                "scheme": "spdx",
-                "identifier": "cc-by-4.0",
+                "id": "cc-by-4.0",
```

Comment:
@lnielsen revisit this when tackling vocabularies... tests/serializers need to be checked